charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -182,8 +182,20 @@ config:
   # -- It allows defining scrape jobs for Kubernetes in a simple way.
   # @default -- See `values.yaml`
   kubernetes:
-    jobs:
-    # By default all pods and endpoints with `prometheus.io/scrape: true` annotation will be discovered.
+    # NewRelic provides a list of Dashboards, alerts and entities for several Services. The integrations_filter configuration
+    # allows to scrape only the targets having this experience out of the box.
+    # If integrations_filter is enabled, then the jobs scrape merely the targets having one of the specified labels matching
+    # one of the values of app_values.
+    # Under the hood, a relabel_configs with action 'keep' are generated, consider it in case any custom extra_relabel_config is needed.
+    integrations_filter:
+      # -- enabling the integration filters, merely the targets having one of the specified labels matching
+      #    one of the values of app_values are scraped. Each job configuration can override this default.
+      enabled: true
+      # -- source_labels used to fetch label values in the relabel config added by the integration filters configuration
+      source_labels: ["app.kubernetes.io/name", "app.newrelic.io/name"]
+      # -- app_values used to create the regex used in the relabel config added by the integration filters configuration
+      app_values: ["redis", "traefik", "calico", "nginx", "coredns", "etcd"]
+
```

Comment:
I would add here also "prometheus-redis-exporter"

Reply:
The regex in un-anchored therefore "redis" will match as well "prometheus-redis-exporter". Should I clarify it with a comment?

Reply:
Also the dashboard has it as "%redis%"

Reply:
I added a clarification, I'll wait for more changes to push it